cloud-db/queries/android.md

```diff
@@ -102,6 +102,55 @@ Besides the operations shown above, the following list out all operations suppor
 
 ## Relational Queries
 
+This example shows how to query all notes (`Note` record) who has an `account` field reference to a user record. In this example, we will query all notes where `account` equals to the current user.
+
+```java
+Record currentUser = Container.defaultContainer(this).getAuth().getCurrentUser(); // Get the current user
+Query noteQuery = new Query("Note").equalTo("account", currentUser.getId());
```

Comment:
Can we do .equalTo("account", currentUser) actually?

Reply:
@chpap No, it doesn't work that way so I have to specify `.id` here.